playwright-e2e/functional/helpers/CaseDataHelper.ts

```diff
@@ -70,10 +71,42 @@ export class ExpressPilotHelper {
   }
 
   static async createContestedPaperCaseWithExpressPilotEnrolled(): Promise<string> {
-    return await this.createCaseWithExpressPilot(      
+    return await this.createCase (      
+      config.caseWorker.email,
+      config.caseWorker.password,
+      './playwright-e2e/data/payload/contested/paper_case/ccd-contested-base.json',
+      'FinancialRemedyContested', 
+      'FR_newPaperCase', 
+      true);
+  }
+
+  static async createContestedFromAWithExpressPilotEnrolled(): Promise<string> {
+    return await this.createCase(      
+      config.applicant_solicitor.email,
+      config.applicant_solicitor.password,
+      './playwright-e2e/data/payload/contested/forma/ccd-contested-base.json',
+      'FinancialRemedyContested',
+      'FR_solicitorCreate',
+      true);
+  }
+
+  static async createBaseContestedFromA(): Promise<string> {
+    return await this.createCase(      
+      config.applicant_solicitor.email,
+      config.applicant_solicitor.password,
+      './playwright-e2e/data/payload/contested/forma/ccd-contested-base.json',
+      'FinancialRemedyContested',
+      'FR_solicitorCreate'
+    );
+  }
+
+  static async createBaseContestedPaperCase(): Promise<string> {
+    return await this.createCase(      
       config.caseWorker.email,
       config.caseWorker.password,
       './playwright-e2e/data/payload/contested/paper_case/ccd-contested-base.json',
-      'FinancialRemedyContested', 'FR_newPaperCase', true);
+      'FinancialRemedyContested', 
+      'FR_newPaperCase'
+    );
```

Comment:
better to put the static texts to a constant maybe. e.g. 'FinancialRemedyContested, 'FR_newPaperCase'